src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,78 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Finding Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Application menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+![Drag Application](/images/locate-launch-applications/drag-application.png)
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop; then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, or from the Terminal.
+
+### The Launcher
+
+Press the `SUPER` key or the Dock Launcher icon to bring up the Launcher.
+
+![Show Launcher](/images/locate-launch-applications/show-launcher.png)
+
+Search for the name of the application.
+
+![Search App Name](/images/locate-launch-applications/search-app.png)
+
+Press `Alt` + `F2` to bring up a command entry window. Enter the name of the application and press `Enter`.
+
+![Command Launcher](/images/locate-launch-applications/command-launcher.png)
+
+### The Dock
+
+Click the application's icon pinned to the Dock.
+
+![Dock Icon](/images/locate-launch-applications/dock-icon.png)
+
+You can pin additional applications to the Dock by right-clicking the application's icon in the Application menu and selecting `Add to Favorites`.
+
+![Add to Favorites](/images/locate-launch-applications/add-to-favorites.png)
+
+Remove applications from the Dock by right-clicking the application and selecting `Remove from Favorites`.
+
+![Remove from Favorites](/images/locate-launch-applications/remove-from-favorites.png)
+
+### The Terminal
+
+Launch the Terminal from the dock; then type the name of the application hit `Enter`.
```

Comment:
The Terminal can also be launched with `Super` + `T`.

Reply:
Added.